optimist.gemspec

```diff
@@ -31,6 +31,6 @@ specify."
 
   spec.add_development_dependency "chronic"
   spec.add_development_dependency "manageiq-style"
-  spec.add_development_dependency "minitest",      "~> 5.4.3"
+  spec.add_development_dependency "minitest",      "~> 5.4"
```

Comment:
Newer versions of minitest work on Ruby 3.4, so unlocking to the y stream allows that newer version to come in.